packages/utils/src/services/contracts/safeContracts.ts

```diff
@@ -26,12 +26,9 @@ export const _getValidatedGetContractProps = (
   }
 }
 export const isMigrationToL2Possible = (safe: SafeState): boolean => {
-  return (
-    safe.nonce === 0 &&
```

Comment:
AFAIR, this check was there to make sure that the indexer works. If nonce is more than 0, i.e. there were transactions made, the indexer refuses to index follow-up txs.

@Uxio0 is it still the case, or can we merge this?